optuna/visualization.py

```diff
@@ -0,0 +1,70 @@
+from optuna.structs import TrialState
```

Comment:
Should we add this module to [index.rst](https://github.com/pfnet/optuna/blob/master/docs/source/reference/index.rst) and create `visualization.rst` file?

Reply:
Thanks! I added.